charts/hedgedoc/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: v2
+name: hedgedoc
+description: A Helm chart for Kubernetes
```

Comment:
:thinking: should be updated from the default

Reply:
done